app/views/world_location_news/show.html.erb

```diff
@@ -49,3 +49,21 @@
     <% end %>
   </div>
 </div>
+
+<% if @world_location_news.mission_statement %>
```

Comment:
This is always present, but can be an empty string, so I don't think this if statement will currently work the way we want it to

Reply:
Ah, I'll update this to check for the empty string.